libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -283,6 +283,15 @@ const AP_Param::GroupInfo AP_TECS::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("HDEM_TCONST", 33, AP_TECS, _hgt_dem_tconst, 3.0f),
 
+    // @Param: STALL_SPEED
+    // @DisplayName: Stall speed used for scaling minimum speed
+    // @Description: If stall prevention is enabled this speed is used to calculate the minimum airspeed while banking. If this is set to 0 then the speed is scaled from the minimum speed. Typically set slightly higher then true stall speed.
+    // @Range: 5.0 75.0
+    // @Units: m/s
+    // @Increment: 1
+    // @User: Advanced
+    AP_GROUPINFO("STALL_SPEED", 34, AP_TECS, _stall_speed, 0),
```

Comment:
nice find!
needs to be an EAS not a TAS to be consistent with other airspeed params, ideally it would be AIRSPEED_STALL and go along with AIRSPEEED_CRUISE etc
also, stall prevention (limiting bank) happens when TECS is not running, so should not be in TECS

Reply:
It is in EAS. When I use it, I multiply by EAS2TAS.

Okay, happy to move where the parameter lives. That is sort of the thought I was having, I only stuck it here as that's what was using it.

Reply:
Param moved, and retested in SITL.